### PR TITLE
doc: Update crosvm links

### PR DESCRIPTION
**Pull request by JonathanWoollett-Light**
## Changes

Updates crosvm links.

## Reason

Links where out of date.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [x] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.
- [x] New `TODO`s link to an issue.
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: tools/bindgen.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 set -eu
 
-# Borrowed from crosvm https://chromium.googlesource.com/chromiumos/platform/crosvm/+/refs/heads/main/tools/impl/bindgen-common.sh#33
+# Borrowed from crosvm https://github.com/google/crosvm+/refs/heads/main/tools/impl/bindgen-common.sh#33
```
Comment (kalyazin) [inline tools/bindgen.sh:13]: this link doesn't work for me

Comment (JonathanWoollett-Light) [reply tools/bindgen.sh]: Fixed.

**Comment by JonathanWoollett-Light** [comment]
> The existing links(https://chromium.googlesource.com/chromiumos/platform/crosvm/+/refs/heads/main/tools/impl/bindgen-common.sh#33) are still valid and the commit message doesn't provide enough justification to update the links The changes are in autogenerated files which doesn't look right and the link in bidgen.sh is broken.

I think we should link the the active crosvm repository in our documentation.

**Comment by kalyazin** [comment]
>I think we should link the the active crosvm repository in our documentation.

I think it would be fair to put that in the commit message, maybe also adding some info on why we believe the original is no longer active.

**Comment by sudanl0** [comment]
> > The existing links(https://chromium.googlesource.com/chromiumos/platform/crosvm/+/refs/heads/main/tools/impl/bindgen-common.sh#33) are still valid and the commit message doesn't provide enough justification to update the links The changes are in autogenerated files which doesn't look right and the link in bidgen.sh is broken.
> 
> I think we should link the the active crosvm repository in our documentation.

sorry I just meant that the commit message was not having the details.

**Comment by JonathanWoollett-Light** [comment]
@kalyazin @sudanl0 I've added to the commit body:

> Updates links in documentation to point to the active crosvm repository.

**Review by sudanl0 — COMMENTED** [review]
How about updating the commit message to:

The message on https://chromium.googlesource.com/chromiumos/platform/crosvm/ says:
`(MOVED) ChromiumOS Virtual Machine Monitor; Development moved to crosvm/crosvm.`
Following this message we update our documentation to point to the active crosvm reposiroty.

**Comment by JonathanWoollett-Light** [comment]
> How about updating the commit message to:
> 
> The message on https://chromium.googlesource.com/chromiumos/platform/crosvm/ says: `(MOVED) ChromiumOS Virtual Machine Monitor; Development moved to crosvm/crosvm.` Following this message we update our documentation to point to the active crosvm reposiroty.

Personally I find `Updates links in documentation to point to the active crosvm repository.` really encapsulates all the infomation I need to know about the commit.

If I where looking at this commit I do not think I would care how the author determined which repository was active.

**Comment by pb8o** [comment]
> > How about updating the commit message to:

> I think commit messages should describe what was done rather than how it was done, and I see this as being part of the "how".

It should also include the "why" it was done. In this case I would think it helps to add this additional detail.

**Comment by JonathanWoollett-Light** [comment]
I think with `Updates links in documentation to point to the active crosvm repository.` the "why" can be inferred that it is because it was pointing at the non-active crosvm repository.

In any case, I have updated the commit message to:

```
Updates links in documentation to point to the active crosvm repository.

https://chromium.googlesource.com/chromiumos/platform/crosvm/ notes:

> (MOVED) ChromiumOS Virtual Machine Monitor;
> Development moved to crosvm/crosvm.

Signed-off-by: Jonathan Woollett-Light <jcawl@amazon.co.uk>
```

**Review by kalyazin — APPROVED** [review]

**Review by sudanl0 — APPROVED** [review]

**Pull request merged**